sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -2302,6 +2302,25 @@
     .setWorstValue(5.0)
     .create();
 
+  /**
+   * @since 6.2
+   */
+  public static final String NEW_RELIABILITY_RATING_KEY = "new_reliability_rating";
+
+  /**
+   * @since 6.2
+   */
+  public static final Metric<Integer> NEW_RELIABILITY_RATING = new Metric.Builder(NEW_RELIABILITY_RATING_KEY, "Reliability Rating on New Code", Metric.ValueType.RATING)
+    .setDescription("Reliability rating on new code")
+    .setDomain(DOMAIN_RELIABILITY)
+    .setDirection(Metric.DIRECTION_WORST)
+    .setDeleteHistoricalData(true)
+    .setOptimizedBestValue(true)
```

Comment:
In fact yes, it's already the case on other ones ? 
I was wrong when we've talked about that